### DISCUSSION TITLE
Is there a way to test a babel plugin online?

### DISCUSSION THREAD

**Discussion by Kreijstal** [babel/babel#15347] [Q&A] [answered]
You can already test published babel plugins, in babel.io, is there a way to try out plugin writing online, or to try writing the babel plugins source code in the interface directly without using one published on npm?

**Comment by nicolo-ribaudo** [comment, marked as answer]
Yes! https://astexplorer.net/

Make sure to choose JavaScript as the language, `@babel/parser` as the parser and babelv7 as the transform.
